bazarr/update_db.py

```diff
@@ -99,6 +99,7 @@
     try:
         c.execute('alter table table_shows add column "year" "text"')
         c.execute('alter table table_shows add column "alternateTitles" "text"')
+        c.execute('alter table table_shows add column "tags" "text"')
 
```

Comment:
You cant just add thus line here... When you already have have year and alternativeTitle in your database it will make an exception and skip other lines... So when i will update it I'll miss your column in db.

This will work only in new installations.

Reply:
Ah, thanks! That shouldn’t need to be there anymore, i put it in when originally testing but it doesn’t use the tags from the database now so I’ll strip that out.